elucidate-server/src/main/java/com/digirati/elucidate/web/converter/AbstractMessageConverter.java

```diff
@@ -61,8 +61,14 @@ protected JSONLDProfile getJsonLdProfile(MediaType contentType, String... defaul
         JSONLDProfile jsonLdProfile = new JSONLDProfile();
 
         String profile = contentType.getParameter("profile");
+ 
         if (StringUtils.isNotBlank(profile)) {
-            defaultContexts = new String[] {profile};
+            if(profile.indexOf(",") > 0){
```

Comment:
The logic here to determine if `profile` is comma separated, and then the logic split it, is overly complicated. I suggest using Apache Commons Lang to make this easier. Swap out lines 66 through 71 for:

`defaultContexts = StringUtils.split(profile, ",");`